.github/workflows/flow-linux-x86.yml

```diff
@@ -185,12 +185,6 @@ jobs:
         shell: bash
         run: |
           ./install_script.sh 
-      - name: Setup Python for testing
```

Comment:
How removing it fixes anything?

Reply:
It is installing a wrong glibc version which then breaks the next step, installing python dependencies (specifically, libpython3.9 cannot find the correct glibc version).

This step was added erroneously not long ago and is already removed from master, 8.0, 1.12, and 1.10.